care/users/api/viewsets/users.py

```diff
@@ -158,15 +158,9 @@ def add_user(self, request, *args, **kwargs):
         serializer = UserCreateSerializer(
             data={**request.data, "password": password}, context={"created_by": request.user},
         )
-        serializer.is_valid(raise_exception=True)
-        username = request.data["username"]
-        if User.objects.filter(username=username).exists():
```

Comment:
you sure this validation is being performed regardless?

Reply:
yes it does as UserCreateSerializer is a modelserializer for User model where username field is unique